packages/rs-sdk/src/platform/document_query.rs

```diff
@@ -209,16 +210,18 @@ impl TryFrom<DocumentQuery> for platform_proto::GetDocumentsRequest {
         let order_by = serialize_vec_to_cbor(dapi_request.order_by_clauses.clone())?;
         // Order clause
 
-        Ok(GetDocumentsRequestV0 {
-            data_contract_id: dapi_request.data_contract.id().to_vec(),
-            document_type: dapi_request.document_type_name.clone(),
-            r#where: where_clauses,
-            order_by,
-            limit: dapi_request.limit,
-            prove: true,
-            start: dapi_request.start.clone(),
-        }
-        .into())
+        //todo: transform this into PlatformVersionedTryFrom
```

Comment:
Why?

You don't need specialized trait like `PlatformVersionedTryFrom` if you convert from specialized struct into more generic one (eg. from GetDocumentsRequestV0 to GetDocumentsRequest). There is no point where we could use version info in this conversion.

And, thanks to strong typing, I don't see any use of PlatformVersionedTryFrom when going from generalized to specialized (GetDocumentsRequest -> GetDocumentsRequestV0). If we use `impl TryFrom<GetDocumentsRequest> for GetDocumentsRequestV0`, the compiler will imply expected version based on the context in which you use this. 

(This comment only applies to dapi-grpc types, night be different in drive/dpp internals)